isis/tests/StretchBlobTest.cpp

```diff
@@ -0,0 +1,57 @@
+#include "CubeStretch.h"
+#include "StretchBlob.h"
+#include "IException.h"
+#include "Cube.h"
+#include "TestUtilities.h"
+#include "Fixtures.h"
+
+#include <QString>
+#include <QDebug> 
+
+#include <gtest/gtest.h>
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+TEST(StretchBlob, Constructors) {
+  // Default
+  StretchBlob stretchBlob;
+
+  // Set Name
+  StretchBlob nameStretchBlob("name");
+
+  // Set Stretch
+  CubeStretch cubeStretch("CetStretch", "defaultType", 1);
+  StretchBlob stretchStretchBlob(cubeStretch);
+
+  EXPECT_STREQ(stretchBlob.Name().toLatin1().data(), "CubeStretch");
+  EXPECT_STREQ(stretchBlob.Type().toLatin1().data(), "Stretch"); 
+
+  EXPECT_STREQ(nameStretchBlob.Name().toLatin1().data(), "name");
+  EXPECT_STREQ(nameStretchBlob.Type().toLatin1().data(), "Stretch"); 
+
+  EXPECT_STREQ(stretchStretchBlob.Name().toLatin1().data(), "CubeStretch");
```

Comment:
Base on line 24 setting the name of cubeStretch to "CetStretch", wouldn't this fail?

Reply:
Thanks for catching this! I wonder if this was a strange typo I managed to mash in-between testing and pushing the change.